docs/layers/s2-layers/s2-layer.md

```diff
@@ -0,0 +1,93 @@
+<p class="badges">
+  <img src="https://img.shields.io/badge/64--bit-support-blue.svg?style=flat-square" alt="64-bit" />
+  <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="extruded" />
+</p>
+
+# S2Layer (Experimental)
+
+The S2Layer renders filled and/or stroked polygons, with geometry automatically calculated based on an S2 token (geospatial index).
+
+For more information
+
+```js
+import DeckGL, {PolygonLayer} from '@deck.gl/s2-layers';
+
+const App = ({data, viewport}) => {
+
+  /**
+   * Data format:
+   * [
+   *   {
+   *     // S2 Cell in SF Bay Area
+   *     token: "80858004",
+   *     // Simple polygon (array of coords)
+   *     token: [[-122.4, 37.7], [-122.4, 37.8], [-122.5, 37.8], [-122.5, 37.7], [-122.4, 37.7]],
```

Comment:
I think this comment is a little confusing. The token is passed to the S2 library which generates that polygon. Can we rephrase a little, or cut the polygon?